clients/python/test/e2e/ci/bash/osparc_version.bash

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -o errexit  # abort on nonzero exitstatus
+set -o nounset  # abort on unbound variable
+set -o pipefail # don't hide errors within pipes
+
+version=$1
+report=$(mktemp)
+trap 'rm -rf ${report}' ERR
+
+if [ "${version}" == "latest_release" ]; then
+    pip install osparc --force-reinstall --dry-run --quiet --report "${report}"
```

Comment:
THOUGHT:
this requires `pip` and `jq` being installed. For the future, it would be nice to reduce all dependencies to `docker`

This should be resolved by having a devcontainer that installs `uv`, `jq`, `python` etc ... (basically whatever is in `make info-tools`

Reply:
cannoit use `uv pip install` here?